Fix docstring on optuna/integration/tensorflow.py

nuka137 opened the pull request:
This PR fixes docstring on optuna/integration/tensorflow.py.

From https://github.com/optuna/optuna/issues/734#issuecomment-597459320 , I removed original docstring and put a reference to examples/pruning/tensorflow_estimator_integration.py.
I think examples/pruning/tensorflow_estimator_integration.py is simple enough to understand a usage.

Related Issue: #734

toshihikoyanase commented on optuna/integration/tensorflow.py:
```diff
@@ -20,23 +20,9 @@ class TensorFlowPruningHook(SessionRunHook):
-                    eval_spec
-                )
+        See example on GitHub for adding a pruning SessionRunHook for a TensorFlow's Estimator.
+        https://github.com/optuna/optuna/blob/master/examples/tensorflow_estimator_simple.py
```
How about using the link instead of showing the URL?

![image](https://user-images.githubusercontent.com/3255979/76960510-f7980980-695e-11ea-9edc-23dccf8d4762.png)

Integration modules for XGBoost and Scikit-Optimize employ the links, so we can follow the convention.

![image](https://user-images.githubusercontent.com/3255979/76960533-02eb3500-695f-11ea-8821-7dca8702a556.png)

![image](https://user-images.githubusercontent.com/3255979/76960585-18605f00-695f-11ea-9740-95c2bbcda636.png)


And also, I think we can simplify the sentence because the words like TensorFlow and SessionRunHook are shown in the previous sentence.

The following is just an example:

> See [an example on GitHub](https://github.com/optuna/optuna/blob/master/examples/tensorflow_estimator_simple.py) to check the usage of the pruning hook.

himkt replied:
I think it would be good to include the commit hash in the link since
a path to an example may change in the future.
(e.g. `https://github.com/optuna/optuna/blob/30dba8814d332ff40d9c7465ee660a8ecf499547/examples/tensorflow_estimator_simple.py`)

For me, it also looks good not to simplify the words. 👍 
If we hold these words, it's reasonable for me to avoid repeatedly use `for`.
Let me put an example as well:

> See [the example](https://github.com/optuna/optuna/blob/30dba8814d332ff40d9c7465ee660a8ecf499547/examples/tensorflow_estimator_simple.py) if you want to add a pruning SessionRunHook for TensorFlow's Estimator.

nuka137 commented:
@toshihikoyanase @himkt 

Thanks!
I fixed the issues you mentioned.

Fixed code will generate below HTML.

![optuna_doc](https://user-images.githubusercontent.com/19800057/77268490-75ef0580-6ce9-11ea-9937-5f509c464690.png)

nuka137 commented:
@toshihikoyanase 

Rebased now.
I think the CI testing seems to be passed now.

toshihikoyanase commented:
@himkt Thanks! Merged.
